conf/machine/include/imx-base.inc

```diff
@@ -234,6 +234,18 @@ MACHINE_FIRMWARE_append_use-mainline-bsp = " linux-firmware-imx-sdma-imx6q linux
 
 MACHINE_EXTRA_RRECOMMENDS += "${MACHINE_FIRMWARE}"
 
+# Below SECO names are used by imx-boot and imx-seco recipes,
+# therefore they are extracted into a separate include file.
+#
+# NOTE: SECO_FIRMWARE_NAME defaults to UNSUPPORTED, which breaks
+# the build if the machine does not have a SECO support.
+SECO_FIRMWARE_NAME              ?= "UNSUPPORTED"
+SECO_FIRMWARE_NAME_mx8qm         = "mx8qmb0-ahab-container.img"
+SECO_FIRMWARE_NAME_mx8qxp        = "mx8qxb0-ahab-container.img"
+SECO_FIRMWARE_NAME_mx8qxpc0      = "mx8qxc0-ahab-container.img"
+SECO_FIRMWARE_NAME_mx8phantomdxl = "mx8qxb0-ahab-container.img"
+SECO_FIRMWARE_NAME_mx8dxl        = "mx8dxla0-ahab-container.img"
```

Comment:
```suggestion
# Below SECO names are used by imx-boot and imx-seco recipes.
SECO_FIRMWARE_NAME              ?= ""
SECO_FIRMWARE_NAME_mx8qm         = "mx8qmb0-ahab-container.img"
SECO_FIRMWARE_NAME_mx8qxp        = "mx8qxb0-ahab-container.img"
SECO_FIRMWARE_NAME_mx8qxpc0      = "mx8qxc0-ahab-container.img"
SECO_FIRMWARE_NAME_mx8phantomdxl = "mx8qxb0-ahab-container.img"
SECO_FIRMWARE_NAME_mx8dxl        = "mx8dxla0-ahab-container.img"
```

Reply:
On both recipes using those, we should add something like:

```python
python () {
    seco_firmware = d.getVar("SECO_FIRMWARE_NAME", False)
    if not seco_firmware:
        raise bb.parse.SkipRecipe("The 'SECO_FIRMWARE_NAME' must be defined for the SoC in 'imx-base.inc'.")
}
```

I think it does what we need.